src/components/Channel/PlayerContainer.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import styled from '@emotion/styled';
+import usePlayers from '../../hooks/channel/usePlayers';
+
+import PlayerItem from './PlayerItem';
+import ErrorBox from '../ErrorBox';
+
+export default function PlayerContainer() {
+  const {
+    query: { channelId },
+  } = useRouter();
+  const { players, error } = usePlayers(channelId);
+
+  if (error) {
+    return <ErrorBox message={error.message} />;
+  }
+
+  return (
+    <Wrapper>
+      {players.map((user) => (
+        <PlayerItem key={user} player={user} />
```

Comment:
```suggestion
      {players.map((player) => (
        <PlayerItem key={player} player={player} />
```

1. 사소하지만 players 배열을 map하는거니까 변수명 user보다는 `player`가 낫지 않을지..
2. `user`는 객체 아닌가요? key값에 `user`가 그대로 들어가는게 좀 이상하네요. `user`가 객체면 그 안에서 unique 한 값(e.g. `user._id`??) 이런걸로 넣어주세요.

Reply:
앗 넵 수정하겠습니다!